README.md

```diff
@@ -21,7 +21,7 @@ recommend using the [app skeleton](https://github.com/cakephp/app) as
 a starting point. For existing applications you can run the following:
 
 ``` bash
-$ composer require cakephp/cakephp:"~3.2"
+$ composer require cakephp/cakephp:"~3.3"
```

Comment:
A recent blog post about this topic:
https://developer.happyr.com/always-use-caret-instead-of-tilde

Personally, I try to always use the tilde...